fuel-tx/src/transaction/validity.rs

```diff
@@ -84,26 +86,31 @@ impl Input {
         witnesses: &[Witness],
         parameters: &ConsensusParameters,
     ) -> Result<(), CheckError> {
+        // TODO: Add verification of the `message_id` with hash from `Message`'s fields.
```

Comment:
It will be done in the upcoming PR. It awaits an answer to the [question](https://github.com/FuelLabs/fuel-specs/pull/443#discussion_r1115531391).